### DISCUSSION TITLE
Change button color inside Create/Edit.. not modal

### DISCUSSION THREAD

**Discussion by valpuia** [filamentphp/filament#6533] [Help] [answered]
Using `->color('primary')` i can change the listpage button, like `add new`, `edit`, `delete` etc.. but inside the create and edit page, I cannot find the way to change the button color for `create` and `cancel`..

Is there an easy way to change the button color inside PAGE

**Comment by danharrin** [comment, marked as answer]
You can override `getSaveFormAction()` and `getCancelFormAction()` etc on those pages

**Reply by valpuia** [reply]
Ok...a little change is needed

For create page => `getCreateFormAction()`
For update page => `getSaveFormAction()`
Cancel is same `getCancelFormAction()`

Thanks @danharrin

**Comment by Vithyasagar-k** [comment]
I have default white color in filament admin page how to change @danharrin
